src/services/admin.ts

```diff
@@ -0,0 +1,42 @@
+import { EmployeeModel } from '../models/employee'
+import { AdminDashboardData } from '../types/admin'
+import { CreateComment } from '../types/comment'
+import { Employee } from '../types/employee'
+import { CommentService } from './comment'
+import { LogService } from './log'
+import { Log } from '../types/log'
+
+const signin = async (email: string, password: string): Promise<Employee | null> => {
+    const user: Employee = await EmployeeModel.findOne({ email })
+    if (!user || user.password !== password || !user.isAdmin) {
+        return null
+    }
+    return user
+}
+
+
+const getDashboard = async (): Promise<AdminDashboardData> => {
+    const logs: Log[] = await LogService.find()
+    return { logs }
+}
+
+const getLog = async (logId: number): Promise<Log> => {
+    return await LogService.findOne(logId)
+}
+
+const createComment = async (comment: string, commentedBy: number, logId: number): Promise<boolean> => {
+    const createComment: CreateComment = {
+        comment,
+        commentedOn: new Date(),
```

Comment:
I think new Date() will return value in `Mon Sep 20 2021 13:34:19 GMT+0545 (Nepal Time)` please format it